package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
```

Comment:
(Oh, and I was only asking because I wanted your advisement! Thanks for the detail!)

I just noticed, and we can hit this in a future patch for sure, `dev:*` should be escaped to avoid possible shell completion.